src/Pages/saranglee/Main/Main.js

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from 'react';
+import { withRouter } from 'react-router-dom';
+
+import Nav from './Components/Nav/Nav';
+import Feed from './Components/Feed/Feed';
+import Story from './Components/Story/Story';
+import Recommendation from './Components/Recommendation/Recommendation';
+
+import '../../../Styles/reset.scss';
+import '../../../Styles/common.scss';
+import '../common.scss';
+import './Main.scss';
+
+class MainSarang extends Component {
+  constructor() {
+    super();
+    this.state = {
+      feedData: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data_saranglee/feedData.json', {
+      method: 'GET'
+    })
```

Comment:
```suggestion
    fetch('/data_saranglee/feedData.json')
```
- 자기 자신의 public에 접근할 때는 위와 같이 주소를 생략할 수 있습니다.
- get method는 기본값이기 때문에 생략해줄 수 있습니다.

Reply:
넵 수정해놓겠습니다~ 감사합니다!